docs/guides/operator-guide-azure-enclave.md

```diff
@@ -9,129 +9,310 @@ sidebar_position: 18
+When the registration process is complete, you'll receive the following:
 
-### Steps
+- An operator key, exclusive to you, that identifies you with the UID2 service as a Private Operator. During configuration, use this as the value for `OPERATOR_KEY`. This value is both your unique identifier and a password; store it securely and do not share it.
 
-1. Make sure the operator service is built using the `azure` profile:
+  :::note
+  You'll receive a separate operator key for each deployment environment.
+  :::
 
-```
-# From the root source folder
-export enclave_platform=azure-sgx
-./setup_dependencies.sh
-mvn package -P azure
-```
+- A link to the UID2 Private Operator for Azure GitHub release page. For example: [https://github.com/IABTechLab/uid2-operator/releases/tag/v5.20.39-SNAPSHOT-azure-cc](https://github.com/IABTechLab/uid2-operator/releases/tag/v5.20.39-SNAPSHOT-azure-cc).
 
-2. While in the `scripts/azure` directory, choose one of the operator config files to include
-in the enclave. The configs are available under `conf/` folder and you should omit the `-config.json`
-suffix.
+### Install Azure CLI
 
-3. Build the enclave and the docker image and specify the config file to use. For example:
+Install the Azure command-line interface. See [How to install the Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) in the Azure documentation.
 
-```
-./build.sh prod
-```
-### Artifacts
+### Get the Required Azure Permissions
+
+You'll need to have subscription owner permission so that you can create a resource group.
 
-The build script produces a few artifacts:
+When that's done, you only need contributor permission on the resource group level for that resource.
 
- - `ghcr.io/iabtechlab/uid2-operator-azure-occlum:dev` -- docker image containing occlum and the UID2 operator service enclave.
- - `build/uid2-operator-azure-sgx.tar.gz` -- tarball of the docker image above.
- - `build/uid2-operator/uid2-operator.tar.gz` -- tarball of the occlum enclave package.
- - output of `sgx_quote` application within the container -- this verifies that the occlum enclave can be launched and gives you basic details about the enclave (MRSIGNER, MRENCLAVE, PRODID, SVN).
+For details, see [Azure roles](https://learn.microsoft.com/en-us/azure/role-based-access-control/rbac-and-directory-admin-roles#azure-roles) in the Azure documentation.
 
-## Test Run
+When all prerequisite steps are complete, you're ready to deploy the UID2 Private Operator. See [Deployment](#deployment).
 
-You can test run the built image locally on the build box:
+## Deployment Environments
 
- - `./run.sh` -- starts the operator service
- - `./run.sh occlum run /bin/sgx_quote` -- dump basic information about the enclave
+The following environments are available. As a best practice, we recommend that you test and verify your implementation in the integration environment before deploying in the production environment.
+
+:::note
```

Comment:
This syntax is different from what we have in other guides which use >Note. I am not sure if there is another step in the process that fixes this, but when I look at the preview, this is not rendering correctly